src/detect-http-server.c

```diff
@@ -33,6 +33,7 @@
  * used by the origin server to handle the request.
  */
 
+#define KEYWORD_NAME_LEGACY "http.server"
```

Comment:
I guess this doesn't make sense, leave it out and add some ifdefs to the template?

Reply:
Right ... and the same applies to `detect-http-location.c

Reply:
Yes indeed.

Reply:
Will remove from detect-http-{server,location}.c